### ISSUE TITLE
Folders in columns view cannot be deselected & do not show checkboxes

### ISSUE THREAD

**Issue by OscarDL** [files-community/Files#12969]
### Description

When using columns view in a directory, once a subfolder in this view is selected, it's impossible to deselect it unless going inside another subfolder.

In parallel to this, we can see that there are no checkboxes for items in columns view, despite me having this option toggled on in the settings, as you can see it working before going in the columns view folder (let me know if I should split this in another issue).

[GIF](https://i.imgur.com/Gje2RZD.gif)

### Steps To Reproduce

1. Enable columns view for a specific folder
2. Click on a folder when in columns view to select it
3. Click on an empty space to deselect it (this is what fails)

### Requirements

The selected folder should be deselected when clicking on an empty space, just like it does in other viewing modes (details, cards, etc.)

### Files Version

2.5.10.0

### Windows Version

Windows 11 Pro 22H2 22621.1992

### Log File

No relevant logs to share

**Comment by yaira2** [comment]
Thank you for reporting this issue.
